internal/debug/flags.go

```diff
@@ -242,3 +254,28 @@ func Exit() {
 		closer.Close()
 	}
 }
+
+func getLogLocationURL(ctx *cli.Context) (*url.URL, error) {
+	locationURL, err := url.Parse(ctx.String(logLocationFlag.Name))
+	if err == nil {
```

Comment:
```suggestion
	if err != nil {
		return nil, err
	}
```
Exit early and un-indent the clause